### ISSUE TITLE
User Guide is not clear when advising: 'pick an SDK'

### ISSUE THREAD

**Issue by i-s-o-g-r-a-m** [cursiveclojure/cursive#1086]
The User Guide's 'Leiningen' page says:

> Pick an SDK, then confirm your project name and file locations.

This isn't much help to someone setting up a fresh install of IntelliJ, where there are not any SDKs available -- one doesn't know which SDK to get (and then pick). It's a real stumbling block. Would it be possible to flesh this part out a bit more: which SDK are we wanting in this situation?

Thanks!

**Comment by cursiveclojure** [comment]
Yes, this is difficult for users new to IntelliJ and I'm definitely planning to clarify it. This is the best information available right now: https://cursiveclojure.com/archive/689.html, and I'm planning to integrate that into the official documentation.

**Comment by i-s-o-g-r-a-m** [comment]
Great, thanks!
